regression-tests.dnsdist/test_SNI.py

```diff
@@ -39,7 +39,7 @@ class TestSNI(DNSDistTest):
     _config_params = ['_testServerPort', '_tlsServerPort', '_serverCert', '_serverKey', '_dohWithNGHTTP2ServerPort', '_serverCert', '_serverKey', '_doqServerPort', '_serverCert', '_serverKey', '_doh3ServerPort', '_serverCert', '_serverKey', '_serverName']
 
     # enable these once Quiche > 0.22 is available, including https://github.com/cloudflare/quiche/pull/1895
```

Comment:
Well maybe consider removing/updating that content, then?
```suggestion
    # enable these once Quiche > 0.22 is available, including https://github.com/cloudflare/quiche/pull/1895
```